Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -34,7 +35,18 @@ public struct GestureOptions: Equatable {
 
     /// Whether pan is enabled for the pinch gesture.
     /// Defaults to `true`.
-    public var pinchPanEnabled: Bool = true
+    ///
+    /// Setting this property to `true` sets ``GestureOptions/focalPoint`` to `nil`.
+    public var pinchPanEnabled: Bool = true {
+        didSet {
+            if focalPoint != nil, pinchPanEnabled {
+                Log.warning(
+                    forMessage: "Gesture options misconfiguration detected: pinch pan cannot be enabled when having a fixed focal point. Focal point option will be set to nil.",
```

Comment:
if the default value for the `pinchPanEnabled` is true, what if the user set the focal point later when the `pinchPanEnabled` is already enabled?

Is there anyway we can handle it internally without changing `focalPoint` to nil when enabling `pinchPanEnabled`?

Reply:
Resolved according to https://github.com/mapbox/mapbox-maps-ios/pull/1122#discussion_r812119694